scripts/manualAuditDataRequests/initiateCopyAndDecrypt/isDatesArray.test.ts

```diff
@@ -0,0 +1,48 @@
+import { isDatesArray } from './isDatesArray'
+
+describe('isDatesArray', () => {
+  it('should return true if the array is an array of dates', () => {
+    const testArray = ['2020-01-01', '2020-01-02']
```

Comment:
Minor point, but you could inline all these variables, there's no need to declare something you're only using once, unless it adds clarity, which in this case I don't think it does.

Reply:
Done